### PR TITLE
ZENKO-3055 Add a changelog

### PR DESCRIPTION
**Pull request by nicolas2bert**

### CODE REVIEW

**Review by anurag4DSB — APPROVED** [review]
I did randomly check of these tickets. 
Question: do we need a change log for all projects?

**Comment by nicolas2bert** [comment]
@anurag4DSB Yes, please check with @terrydervaux.

**Review by tmacro — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Change Log
+
+## [0.1.0] - 2021-01-07
+
+Initial release
```
Comment (miniscruff) [inline CHANGELOG.md]: No need to include the initial release description if we also include what was added, granted I feel the "Added" header is unnecessary here.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Change Log
+
+## [0.1.0] - 2021-01-07
```
Comment (miniscruff) [inline CHANGELOG.md]:
```suggestion
## 0.1.0 - 2021-01-07
```
No need for brackets if there is no link to the tagged release.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Change Log
```
Comment (miniscruff) [inline CHANGELOG.md]:
```suggestion
# Changelog
```
One word

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+- ZENKO-2732 List locations
+- ZENKO-2731 Create location
+- ZENKO-2709 Delete account
+- ZENKO-2701 List accounts
+- ZENKO-2675 Account creation
+- ZENKO-2644 OAuth 2.0 User authentication
+
+### Changed
+
+### Fixed
```
Comment (miniscruff) [inline CHANGELOG.md]:
```suggestion
```
No need for empty sections

**Review by miniscruff — CHANGES_REQUESTED** [review]

**Review by miniscruff — APPROVED** [review]

**Review by terrydervaux — APPROVED** [review]
LGTM.

**Comment by miniscruff** [comment]
@nicolas2bert we might want to update package.json version to match the changelog

**Comment by nicolas2bert** [comment]
@bert-e approve

**Pull request merged**